dev/env/scripts/lib.sh

```diff
@@ -315,7 +318,21 @@ docker_logged_in() {
 }
 
 load_external_config() {
-  local chamber="${CHAMBER:-$CHAMBER_DEFAULT}"
+  local chamber
+  local use_aws_vault="${USE_AWS_VAULT:-$USE_AWS_VAULT_DEFAULT}"
+  if [ "$use_aws_vault" = true ]; then
```

Comment:
nit: consistency. I prefer `[[ ]]` because it has more features for string matching.

```suggestion
  if [[ "$use_aws_vault" = true ]]; then
```

Reply:
Fixed